_data/simple-icons.json

```
@@ -15170,6 +15170,16 @@
             "hex": "FF0000",
             "source": "https://partnermarketinghub.withgoogle.com/#/brands"
         },
+        {
+            "title": "YouTube Shorts",
+            "hex": "FF0000",
+            "source": "https://commons.wikimedia.org/wiki/File:Youtube_shorts_icon.svg",
```

Source

---

So, the icon is available on https://www.youtube.com/shorts
as
```
<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;"><path d="m17.77 10.32-1.2-.5L18 9.06c1.84-.96 2.53-3.23 1.56-5.06s-3.24-2.53-5.07-1.56L6 6.94c-1.29.68-2.07 2.04-2 3.49.07 1.42.93 2.67 2.22 3.25.03.01 1.2.5 1.2.5L6 14.93c-1.83.97-2.53 3.24-1.56 5.07.97 1.83 3.24 2.53 5.07 1.56l8.5-4.5c1.29-.68 2.06-2.04 1.99-3.49-.07-1.42-.94-2.68-2.23-3.25zM10 14.65v-5.3L15 12l-5 2.65z"></path></svg>
```

Original source will take precedence in this case.
```suggestion
            "source": "https://www.youtube.com/shorts",
```